README.md

```diff
@@ -17,9 +17,11 @@ The carbon intensity of each country is measured from the perspective of a consu
 The carbon intensity of each type of power plant takes into account emissions arising from the whole lifecyle of the plant (construction, fuel production, operational emissions, and decomissioning). GHG intensity factors used in the map are detailed in [co2eq-parameters.js](https://github.com/corradio/electricitymap/blob/master/shared/co2eq_parameters.js). These numbers come from the following scientific peer reviewed litterature: 
 - IPCC 2014 Assessment Report is used as reference in most instances (see a summary in the [wikipedia entry](https://en.wikipedia.org/wiki/Life-cycle_greenhouse-gas_emissions_of_energy_sources#2014_IPCC.2C_Global_warming_potential_of_selected_electricity_sources))
 
-Country specific emissions:
+Country-specific powerplant carbon-intensity factors:
```

Comment:
I would remove "powerplant", because it's more production type than power plant (which is spelled in two words btw)